.devcontainer/experimental/Dockerfile

```
@@ -0,0 +1,40 @@
+FROM openinf/grimesai-salvage-tex:lunar@sha256:85db277df2bb6b07922478085cd612a67a9a7d3923b09135a7f6bd9bd65ca89a
```

Dockerfile not being checked for license blurbs is kinda unlike us

```suggestion
FROM openinf/grimesai-salvage-tex:lunar@sha256:85db277df2bb6b07922478085cd612a67a9a7d3923b09135a7f6bd9bd65ca89a
```